assembly/src/main/jib/app/stream-registry-config.yaml

```diff
@@ -56,10 +56,16 @@ kafkaStreamsConfig:
     replication.factor: ${STREAM_REGISTRY_KSTREAMS_REPLICATION_FACTOR:-1}
     num.stream.threads: 2
 
-# producerTopic name is the name of the underlying Kafka Topic that would store the Streams created by the application.
-# IMPORTANT: make sure the producerTopic is compacted or retention.ms=-1 so that the created Streams are not lost.
+# eventStoreTopic name is the name of the underlying Kafka Topic that would store the Streams created by the application.
+# IMPORTANT: make sure the eventStoreTopic is compacted or retention.ms=-1 so that the created Streams are not lost.
 topicsConfig:
-  producerTopic: _streamregistry_streams${STREAM_REGISTRY_PRODUCER_TOPIC_SUFFIX:-_v0.1}
+  eventStoreTopic:
+    name: _streamregistry_streams${STREAM_REGISTRY_EVENTSTORE_TOPIC_SUFFIX:-_v0.1}
+    partitions: ${STREAM_REGISTRY_EVENTSTORE_PARTITIONCOUNT:-1}
+    replicationFactor: ${STREAM_REGISTRY_EVENTSTORE_REPLICATIONFACTOR:-1}
```

Comment:
The other uses of replication factor are `REPLICATION_FACTOR`. Let's aim for consistency.. 

Same for other properties. Add underscores.